Packages/com.unity.inputsystem/InputSystem/InputSystemObject.cs

```diff
@@ -40,6 +42,50 @@ public void ReviveAfterDomainReload()
             SetUpRemoting();
         }
 
+        private SerializedObject GetPlayerSerializedPlayerSettings()
+        {
+            PlayerSettings[] array = Resources.FindObjectsOfTypeAll<PlayerSettings>();
+            var playerSettings = array.Length > 0 ? array[0] : null;
+
+            if (playerSettings)
+                return new SerializedObject(playerSettings);
+
+            return null;
+        }
+
+        public bool IsNewInputSystemActiveInPlayerSettings()
+        {
+            var serializedPlayerSettings = GetPlayerSerializedPlayerSettings();
+
+            if (serializedPlayerSettings != null)
+                return serializedPlayerSettings.FindProperty("enableNativePlatformBackendsForNewInputSystem").boolValue;
```

Comment:
We could make this somewhat future proof by considering the backends enabled if the property isn't there.